packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -68,55 +69,70 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
     this.callDiagnosticEventsBatcher = new CallDiagnosticEventsBatcher({}, {parent: this.webex});
   }
 
+  /**
+   * Returns the login type of the current user
+   * @returns one of 'login-ci','unverified-guest'
+   */
+  getCurLoginType() {
+    // @ts-ignore
+    if (this.webex.canAuthorize) {
+      // @ts-ignore
+      return this.webex.credentials.isUnverifiedGuest ? 'unverified-guest' : 'login-ci';
+    }
+
+    return null;
+  }
+
   /**
    * Get origin object for Call Diagnostic Event payload.
    * @param options
    * @param meetingId
    * @returns
    */
   getOrigin(options: GetOriginOptions, meetingId?: string) {
-    let defaultClientType: Event['origin']['clientInfo']['clientType'];
-    let defaultSubClientType: Event['origin']['clientInfo']['subClientType'];
-    let environment: Event['origin']['environment'];
-
-    if (meetingId) {
+    const defaultClientType: Event['origin']['clientInfo']['clientType'] =
```

Comment:
it would be nice to extract the type

Reply:
it's already extracted, i just missed to replace it here